azure-functions/acbs-function/mappings/facility/facility-update.js

```diff
@@ -40,18 +40,14 @@
 const helpers = require('./helpers');
 const CONSTANTS = require('../../constants');
 
-const facilityUpdate = (facility, acbsFacility, obligorName) => {
-  const { facilitySnapshot } = facility;
-
-  return {
-    ...acbsFacility,
-    capitalConversionFactorCode: helpers.getCapitalConversionFactorCode(facilitySnapshot),
-    issueDate: helpers.getIssueDate(facility, acbsFacility.effectiveDate),
-    facilityStageCode: CONSTANTS.FACILITY.STAGE_CODE.ISSUED,
-    foreCastPercentage: CONSTANTS.FACILITY.FORECAST_PERCENTAGE.ISSUED,
-    productTypeName: facilitySnapshot.facilityType,
-    obligorName,
-  };
-};
+const facilityUpdate = (facility, acbsFacility, obligorName) => ({
+  ...acbsFacility,
+  capitalConversionFactorCode: helpers.getCapitalConversionFactorCode(facility),
+  issueDate: helpers.getIssueDate(facility, acbsFacility.effectiveDate),
+  facilityStageCode: CONSTANTS.FACILITY.STAGE_CODE.ISSUED,
+  foreCastPercentage: CONSTANTS.FACILITY.FORECAST_PERCENTAGE.ISSUED,
+  productTypeName: facility.facilityType || facility.facilitySnapshot.type,
```

Comment:
I think need to add an extra `facilityType` condition here for it to work in both scenarios, with GEF and BSS?

```js
(facility.facilityType
  || facility.facilitySnapshot.type
  || facility.facilitySnapshot.facilityType)
```

When this is called with submission mapping, all facilities will have `facility.facilityType`. Otherwise - only GEF has `.type`, BSS has `.facilityType`.